caas/kubernetes/provider/admissionregistration_test.go

```diff
@@ -241,7 +241,7 @@ func (s *K8sBrokerSuite) TestEnsureMutatingWebhookConfigurationsUpdate(c *gc.C)
 		c, cfgs,
 		s.mockMutatingWebhookConfiguration.EXPECT().Create(cfg1).Return(cfg1, s.k8sAlreadyExistsError()),
 		s.mockMutatingWebhookConfiguration.EXPECT().
-			List(metav1.ListOptions{LabelSelector: "juju-app==app-name,juju-model==test"}).
+			List(metav1.ListOptions{LabelSelector: "juju-app=app-name,juju-model=test"}).
```

Comment:
Does this have upgrade implications? (are there existing deployments with the "--")

Reply:
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#equality-based-requirement
"=", "=="  represent equality (and are simply synonyms)